### DISCUSSION TITLE
Replace shipped font awesome with other variant

### DISCUSSION THREAD

**Discussion by mpgr04** [fomantic/Fomantic-UI#1349] [General] [answered]
i just wonder if there is a possibility to replace the currently shipped font awesome version with font awesome duotone for example, i read that it is possible in the npm version, but is there also a way in the css only distribution ?

**Comment by lubber-de** [comment, marked as answer]
The [CSS Version](https://github.com/fomantic/Fomantic-UI-CSS) is the precompiled default theme of the LESS repository.
You can use the [LESS repository](https://github.com/fomantic/Fomantic-UI-LESS) instead, change the @fontname in https://github.com/fomantic/Fomantic-UI-LESS/blob/master/themes/default/elements/icon.variables and every icon in https://github.com/fomantic/Fomantic-UI-LESS/blob/master/themes/default/elements/icon.overrides according to your needs and compile fomantic by yourself
